runtime/android/core_internal/src/org/xwalk/core/internal/XWalkSettings.java

```diff
@@ -435,7 +436,17 @@ public boolean getJavaScriptCanOpenWindowsAutomatically() {
             return mJavaScriptCanOpenWindowsAutomatically;
         }
     }
-
+	
+    //to support spatial navigation, like a TV remote control
```

Comment:
Style: you need a space after `//`, start your sentence with a capital letter and end it with a period (".").